Kernel/Problem.cpp

```diff
@@ -98,7 +98,6 @@ void Problem::initValues()
   _mayHaveInequalityResolvableWithDeletion = true;
   _mayHaveXEqualsY = true;
   _propertyValid = false;
-  _property = env.property;
```

Comment:
Is this correct? What did this do before?

Reply:
Could it be you are confusing a deleted line for an added one?

Reply:
Well, actually, you were right - so form of initialization was (as the tests have shown) necessary!